s3/tests/creation.tftest.hcl

```diff
@@ -23,7 +23,7 @@ run "test_bucket_creation" {
   }
 
   assert {
-    condition     = cloudfoundry_service_instance.bucket.service_plan == data.cloudfoundry_service.s3.service_plans[var.s3_plan_name]
+    condition     = cloudfoundry_service_instance.bucket.service_plan == data.cloudfoundry_service_plans.s3.service_plans.0.id
```

Comment:
I think this passing with `basic-sandbox` is a coincidence that depends on that being the first plan listed, isn't it?